app/main.py

```diff
@@ -1 +1,60 @@
-# write your code here
+from typing import List
+
+
+class Animal:
+    """
+    Class representing an animal.
+    """
+    alive: List["Animal"] = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        """
+        Initialize an animal with a name and health.
+        """
+        self.name = name
+        self.health = health
+        self.hidden = False
+        self.__class__.alive.append(self)
```

Comment:
```suggestion
        self.alive.append(self)
```
`__class__` is redundant

Reply:
Fix in other places

Reply:
not fixed